manifests/configmap.yaml

```diff
@@ -10,7 +10,7 @@ data:
 
   debug_logging: "true"
   workers: "4"
-  docker_image: registry.opensource.zalan.do/acid/spilo-cdp-11:1.5-p42
+  docker_image: registry.opensource.zalan.do/acid/spilo-11:1.5-p4
```

Comment:
@sdudoladov , just curious what is the difference between `spilo-cdp-11` and `spilo-11`?

Reply:
`spilo-cdp` is build and pushed by ci/cd system on every commit to the master branch.
Later it is tested and if everything is fine we are tagging it as a `spilo`.

Reply:
is `spilll-11` aligned with Patroni releases or it is for every commit as well, just after test?